src/app/public/modules/repeater/repeater-item.component.scss

```diff
@@ -23,6 +25,7 @@
 .sky-repeater-item-header {
   align-items: center;
   display: flex;
+	padding-top: $sky-padding;
```

Comment:
All of these entries appear to be missing an extra space character before `padding-top`:

```
  display: flex;
 padding-top: $sky-padding;
```

Reply:
I didn't notice this in my IDE, but on closer inspection it looks like they might be tabs instead of spaces! 😲

Reply:
@Blackbaud-ToddRoberts you might want to double check your settings to make sure you're doing 2 spaces per tab.

Reply:
@Blackbaud-SteveBrush this is fixed!